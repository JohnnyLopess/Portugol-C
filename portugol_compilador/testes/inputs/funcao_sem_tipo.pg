programa {
    funcao calcula() {
        inteiro numero = 1
        escreva(numero + 1)
    }
    inicio
        calcula()
}