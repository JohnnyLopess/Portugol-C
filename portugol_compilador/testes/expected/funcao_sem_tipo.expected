#include <stdio.h>
#include <string.h>

void calcula() {
    int numero = 1;
    printf("%d\n", numero + 1);
}

int main(){
    calcula();
    return 0;
}